### PR TITLE
Docs: Add mobile access question to the FAQ

### PR DESCRIPTION
**Pull request by moneymanolis**
Adds "How can I access the Specter Desktop web interface with my phone?" to faq.md.

### CODE REVIEW

**Comment by moneymanolis** [comment]
@relativisticelectron can you review? Feel free to add stuff. I basically just tried to access via mobile to test your PRs.

File: docs/faq.md [hunk 1]
```diff
@@ -399,6 +400,10 @@ Alternatively, you can also define --port 80 if you want to have it on default h
 
 One drawback though is that with http and **external access** you will not get camera scanning functionality. It is an issue if you are using specter-DIY as it's necessary to scan QR codes with signed transactions. To fix that you will need a self-signed certificate, we have a document on that [here](https://github.com/cryptoadvance/specter-desktop/blob/master/docs/self-signed-certificates.md)
 
+## *How can I access the Specter Desktop web interface with my phone?*
+
+Similar as above, add `--host 0.0.0.0` when starting the Specter server (to bind the Specter server to the local IP of the machine that you are running Specter on), thus `python -m cryptoadvance.specter server --host 0.0.0.0`. You can then access the Specter server with your phone by typing `http://LOCAL_IP_OF_MACHINE_RUNNING_SPECTER:25441/` in the phone's browser. Note: Your phone and the machine running Specter have to be connected to the same LAN for this simple approach to work. You might also need to adjust your firewall settings on the machine running Specter. For remote access of your Specter server check the [Tor docs](https://docs.specter.solutions/desktop/tor/).
```
Comment (relativisticelectron) [inline docs/faq.md:405]: Can you put "Note:" in a a new line and make it bold?

**Review by relativisticelectron — COMMENTED** [review]
Looks good to me. That's how I was testing it.

EDIT: Once _spectrum_ is working well, I would really like to see specter running on a mobile phone connected to a remote electrs.  The Web-UI is quite nice and has a faster reaction time than some other phone-wallets.

**Comment by moneymanolis** [comment]
Thanks!

> EDIT: Once _spectrum_ is working well, I would really like to also try specter running on a mobile phone connected to a remote electrs. The specter Web-UI is quite nice and has a faster reaction time than some other phone-wallets.

If you mean that Specter is still running on a laptop etc. and you are just accessing via your phone this will be possible soon I'd say. Native mobile app is a different beast.

**Comment by relativisticelectron** [comment]
> If you mean that Specter is still running on a laptop etc.

No, I mean specter running on a mobile phone (electrum is also python and runs on a mobile phone) and being connected to an electrum server. That would be awesome (I am not sure how important that is though,... remote specter also works well).

**Pull request merged**